### PR TITLE
Adjust the styling and linked the username and profilepics

### PR DESCRIPTION
**Pull request by Kiyoonewton**
Adjust the socialfeed styling link profile pic and username

### CODE REVIEW

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @Kiyoonewton , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Please resolve the conflicts on the PR before requesting for review.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: pages/dashboard/socialfeeds/common/centerFeed.jsx [hunk 1]
```diff
@@ -1,19 +1,19 @@
-import styles from '../../../../styles/search.module.css';
-import CenterCard from './centerCard';
+import styles from '../../../../styles/search.module.css'
+import CenterCard from './centerCard'
 
 const CenterFeed = () => {
```
Comment (micheaol) [inline pages/dashboard/socialfeeds/common/centerFeed.jsx:4]: @Kiyoonewton ,

Please find the error received below:

image

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @Kiyoonewton , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Please also note:
- Please note that some of the links at the footer are not working still. Please look into that ASAP.

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: components/features/dashboard/student/studentProfile/index.jsx [hunk 2]
```diff
@@ -543,7 +545,7 @@ export const ProfilePicture = ({ user }) => {
                   ? user?.user?.profilePhotoUrl
                   : `/assets/img/features/dashboard/student/woman.png`
               }
-              alt="Flowers"
+              alt="ProfilePix"
               style={{
```
Comment (micheaol) [inline components/features/dashboard/student/studentProfile/index.jsx:549]: While you were resolving the conflicts, the hero page image has been tampered with. Kindly fix that ASAP.

## Your design:
image


## Current site:
image


Please make that adjustment and revert ASAP.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @Kiyoonewton , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Kiyoonewton** [comment]
Add the adjustment already

**Pull request merged**